docs/schema.md

```diff
@@ -109,13 +99,12 @@ be `v1beta1` or newer.
 
 The
 [release version of the app](building-deployer.md#tagging-your-deployer-image),
-as a string. This must match the release tag on all your app images. For example,
-`'1.0.5'`.
+as a string. This must match the release tag on all your app images. For example, `'1.0.5'`.
 
 ### `publishedVersionMetadata`
```

Comment:
This entire section might be irrelevant now - @huyhg, could you confirm?

Reply:
Please keep this section.